fspiop/v1_0/openapi3/components/schemas/PartyComplexName.yaml

```diff
@@ -0,0 +1,16 @@
+title: PartyComplexName
+type: object
+description: Data model for the complex type PartyComplexName.
+properties:
+  firstName:
+    type: string
```

Comment:
what about the FirstName type defined above? - it has quite a restricted regexp pattern and here is allowed an unrestricted string